static/fpc/style.css

```diff
@@ -112,5 +110,12 @@ table.title {
 }
 
 body {
-	background-color: black;
+    background-color: black;
```

Comment:
I know spaces are better than tabs, but I prefer consistency :innocent: Can you change the indentation back to tabs? (or change everything to spaces, I guess, but that's a bit more work and pollutes the diff :joy: )